TwitchLib.Client.Models/Internal/IrcMessage.cs

```diff
@@ -45,7 +45,7 @@ public class IrcMessage
         /// <summary>
         /// IRCv3 tags
         /// </summary>
-        public readonly IReadOnlyDictionary<string, string> Tags;
```

Comment:
can't it stay on IReadOnlyDictionary? All other properties/fields are "readonly"

Reply:
It definitely can but it feels like a shame to do so since it has a lot of calls in always-invoked paths. `IrcMessage` is produced internally by `IrcParser` and then consumed internally, never being exposed in any public field or property to the users, even if the type itself is `public` and if it will be ever exposed, it will be done so in an "ownership transfer" way where the user might receive it in event args, where `TwitchClient` no longer cares what happens to it.